pom.xml

```diff
@@ -47,6 +47,17 @@
         <jackson.version>2.13.2</jackson.version>
         <jackson.databind.version>2.13.4.2</jackson.databind.version>
     </properties>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.opentelemetry</groupId>
+                <artifactId>opentelemetry-bom</artifactId>
+                <version>1.39.0</version>
```

Comment:
I have a question here: how is the version control of a specific library done in Java? Does this code mean that user have to use opentelemetry-bom==1.39.0? Is there a way to limit it like >1.0.0, <2.0.0?

Reply:
When compile, Maven while load opentelemetry 1.39.0 to compile the project. But during runtime, it actually works with any compatible dependencies.